mammoth_gazebo/launch/mammoth.launch.py

```diff
@@ -153,6 +155,7 @@ def generate_launch_description():
             ('/lidar/filtered_points', '/mammoth/filtered_points'),
             ('/lidar/raw_scan', '/mammoth/raw_scan'),
             ('/lidar/filtered_scan', '/mammoth/filtered_scan'),
+            ('/lidar/unfiltered_points', '/mammoth/unfiltered_points'),
```

Comment:
Where is this topic coming from? Is there an addition to the lidar_processor I'm missing?